Assets/Scripts/Health.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine.UI; // Required when Using UI elements.
+using UnityEngine;
+
+public class Health : MonoBehaviour
+{
+    [SerializeField]
+    private Image HP;
```

Comment:
comment je variable.

**HP**, is een nietszeggende naam. Noem het bijvoorbeeld: **healthImage** of **heartImage**